pkg/splunk/util/util.go

```diff
@@ -228,6 +230,7 @@ func GetPodExecClient(client splcommon.ControllerClient, cr splcommon.MetaObject
 	return &PodExecClient{
 		client:        client,
 		cr:            cr,
+		name: 		   "splunk",
```

Comment:
Nit: Just curious, is this the only podExecClient used? Is there any other place where we need to update this?